### PR TITLE
Fix JavaGenerator to cleanly emit the latest descriptor.proto.

### PR DESCRIPTION
**Pull request by swankjesse**
We'd been relying on an old descriptor.proto. The new one has a problem with the
comment delimiter. The fix is ugly, but I can't find anything neater.

### CODE REVIEW

File: wire-runtime/src/test/proto-java/com/google/protobuf/EnumValueOptions.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // Code generated by Wire protocol buffer compiler, do not edit.
-// Source file: google/protobuf/descriptor.proto at 389:1
+// Source file: google/protobuf/descriptor.proto at 549:1
```
Comment (swankjesse) [inline wire-runtime/src/test/proto-java/com/google/protobuf/EnumValueOptions.java]: @mattprecious

File: wire-runtime/src/test/proto-java/com/google/protobuf/SourceCodeInfo.java [hunk 2]
```diff
@@ -301,14 +314,72 @@ public Location redact(Location value) {
...
+     *   // Detached comment for corge. This is not leading or trailing comments
+     *   // to qux or corge because there are blank lines separating it from
+     *   // both.
+     *
+     *   // Detached comment for corge paragraph 2.
+     *
+     *   optional string corge = 5;
+     *   /* Block comment attached
+     *    * to corge.  Leading asterisks
+     *    * will be removed. *_/
```
Comment (swankjesse) [inline wire-runtime/src/test/proto-java/com/google/protobuf/SourceCodeInfo.java]: `*_/`

File: wire-runtime/src/test/proto-java/com/google/protobuf/EnumValueOptions.java (hunk 1 above)
Comment (JakeWharton) [inline wire-runtime/src/test/proto-java/com/google/protobuf/EnumValueOptions.java]: karma!

File: wire-runtime/src/test/proto-java/com/google/protobuf/SourceCodeInfo.java (hunk 2 above)
Comment (JakeWharton) [inline wire-runtime/src/test/proto-java/com/google/protobuf/SourceCodeInfo.java]: Interesting. Can we just turn one of the chars into an `&#0NN;`?

**Comment by JakeWharton** [comment]
LGTM

File: wire-runtime/src/test/proto-java/com/google/protobuf/SourceCodeInfo.java (hunk 2 above)
Comment (swankjesse) [inline wire-runtime/src/test/proto-java/com/google/protobuf/SourceCodeInfo.java]: Good idea. Done.

**Pull request merged**